src/Command/Activator/LineItemActivatorCommand.php

```diff
@@ -0,0 +1,221 @@
+
+namespace OAT\SimpleRoster\Command\Activator;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use InvalidArgumentException;
+use OAT\SimpleRoster\Repository\LineItemRepository;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Throwable;
+
+class LineItemActivatorCommand extends Command
+{
+    public const NAME = 'roster:activate:line-item';
+
+    private const AVAILABLE_FIELD_QUERIES = [
+        self::FIELD_ID,
+        self::FIELD_SLUG,
+        self::FIELD_URI,
+    ];
+
+    private const FIELD_ID = 'id';
+    private const FIELD_SLUG = 'slug';
+    private const FIELD_URI = 'uri';
+
+    private const TOGGLE_OPERATIONS = [
+        self::TOGGLE_ACTIVATE,
+        self::TOGGLE_DEACTIVATE
+    ];
+
+    private const TOGGLE_ACTIVATE = 'activate';
+    private const TOGGLE_DEACTIVATE = 'deactivate';
+
+    /** @var LineItemRepository */
+    private $lineItemRepository;
+
+    /** @var SymfonyStyle */
+    private $symfonyStyle;
+
+    /** @var LoggerInterface */
+    private $logger;
+
+    public function __construct(
+        LineItemRepository $lineItemRepository,
+        LoggerInterface $logger
+    ) {
+        parent::__construct(self::NAME);
+
+        $this->lineItemRepository = $lineItemRepository;
+        $this->logger = $logger;
+    }
+
+    protected function configure(): void
+    {
+        parent::configure();
+
+        $this->setDescription('Activate/Deactivate line items into the application');
+        $this->setHelp(
+            <<<'EOF'
+The <info>%command.name%</info> command Activate/Deactivate line items into the application.
+
+    <info>php %command.full_name% <path></info>
+
+To Activate a line item by slug:
+
+    <info>php %command.full_name% activate slug {line-item-slug}</info>
+    
+To Activate a line item by id:
+
+    <info>php %command.full_name% activate id {line-item-id}</info>
+
+To Activate a line item by uri:
+
+    <info>php %command.full_name% activate uri {line-item-id}</info>
+To Deactivate a line item by slug:
+
+    <info>php %command.full_name% deactivate slug {line-item-slug}</info>
+    
+To Deactivate a line item by id:
+
+    <info>php %command.full_name% deactivate id {line-item-id}</info>
+
+To Deactivate a line item by uri:
+
+    <info>php %command.full_name% deactivate uri {line-item-id}</info>
+EOF
+        );
+
+        $this->addArgument(
+            'toggle',
+            InputArgument::REQUIRED,
+            'Accepted two values "activate" to activate a line item. "deactivate" to deactivate a line item.'
+        );
+
+        $fieldQueryDescription = 'How do you want to query the line items that you want to activate/deactivate. '
+            . 'Accepted parameters are: %s';
+        $this->addArgument(
```

Comment:
I don't see a problem here but just for note, but in other commands we are using always options an not arguments. When we had multiple options, we add a check if they are exclusive

This way we would have something like:
-  --i 4
-  --s mySlug
-  --u myUri

What do you think on keep it using `options` instead of arguments`?
Also, wouldn't be better to allow activate/deactivate multiple line items?

Reply:
The reason that I used arguments was that the tree informed values are mandatory. which is not the case for optional parameters, as you told me, I'd need to verify it on the code, but with arguments, you can do it easily. Let me get more feedback from other members. If they also ask for it, I'll change it.
About the multiple line items, it's a good idea. Let me do it.

Reply:
IMO - for query fields, it can be as is. BUT, for activate/deactivate toggle - I'd better use flags (`-d` / `-a`) and presence of one of them would be mandatory (which leads us, indeed, to check in the code). IMO - this way it'd look better